src/Renderer/PolyLineRenderer.js

```diff
@@ -202,15 +500,51 @@ define([
             cs
         );
 
-        var pos3d = vec3.create();
         var origin = vec3.create();
         cs.get3DFromWorld(convertedCoord, origin); // Correction : add [0]
 
         mat4.identity(this.matrix);
         mat4.translate(this.matrix, origin);
 
+        const leftPlane = [-180, 0, 0];
+        const leftNormal = [1, 0, 0];
+        cs.get3DFromWorldInCrs(leftPlane, crsName, this.leftClipPlane);
+        cs.get3DFromWorldInCrs(leftNormal, crsName, this.leftClipNormal);
+
+        const rightPlane = [180, 0, 0];
+        const rightNormal = [-1, 0, 0];
+        cs.get3DFromWorldInCrs(rightPlane, crsName, this.rightClipPlane);
+        cs.get3DFromWorldInCrs(rightNormal, crsName, this.rightClipNormal);
+
+        // Check if some lines are going through the poles
+        // In this case, we want to duplicate the points that do not have the same x coordinate.
+        _fixPoles(lines);
+
+        var normalLines = [];
+        var clippedLines = [];
+
+        var proj = null;
+        if (cs.projection && cs.projection.options && cs.projection.options.projectionName) {
+            proj = cs.projection.options.projectionName;
+        }
+
+        if (cs.flat === true && proj !== Mizar.PROJECTION.Azimuth) {
```

Comment:
Mizar is not defined. It should be Constants , isn't it ?

Reply:
Indeed